pkg/cmd/admin/migrate/migrator.go

```diff
@@ -173,6 +182,9 @@ func (o *ResourceOptions) Complete(f *clientcmd.Factory, c *cobra.Command) error
 					if !others.Has(k) {
 						continue
 					}
+					// TODO: the order here is not deterministic, due to the fact that StringSet is
```

Comment:
Generally, I'd prefer here to use the 'blessed' resource name, iow the groupped one, but that will always end up being 2nd in the sorted list, so I guess it's fine for now.